README.md

```diff
@@ -283,8 +283,9 @@ And finally, you are ready to go! Or at least you should be...
 
 ### Junior Contributors
 
+**Eira Fontanals** | [eirafontanals][eirafontanals] | I am an easy laughter person.
 
-
+[eirafontanals]: <https://github.com/eirafontanals>
```

Comment:
Do what Iris requested below, resolve the two conversations and you are ready to go